aims/aim1.md

```diff
@@ -1,46 +1,48 @@
+
+Single cell data pose unique challenges, as well as opportunities, for deep neural network algorithms.
+Though each estimate of transcript abundance in each cell may include substantially more error than bulk samples, there are also many observations as numerous cells are often assayed.
+Also, the HCA project includes data from multiple cell types, so we expect substantial numbers of observations to be available.
+From our experience with generative deep neural networks [@doi:10.1101/159756], we have found that it can be difficult to predict precisely which advances will enable robust training in a specific context, particularly when the specific context has not been tackled before.
+We describe our standard approach here, as well as a couple selected techniques that we anticipate will be particularly helpful in this setting such as zero-inflated loss and data augmentation, but we expect to also employ other strategies where warranted.
+
+Our standard approach is to first perform an extensive grid search to identify VAE architectures and hyperparameters that are amenable to this context.
+Some parameter settings can be easily ruled out when models fail to train.
+Chris Probert noted on our proposal's GitHub repository [@url:https://github.com/greenelab/czi-rfa/issues/11] that numerous manuscripts demonstrated the advantages of zero-inflated loss in the setting of single cell sequencing data [@doi:10.1186/s13059-015-0805-z @arxiv:1610.05857 @doi:10.1186/s13059-017-1188-0].
+We will evaluate multiple types of reconstruction loss during training.
+These efforts will allow us to identify a subset of parameter combinations that are worth exploring with new approaches designed specifically for this type of genomic data.
+
+For single cell RNA-seq data, we also expect to be able to take advantage of data augmentation for the first time with genomic data.
+Data augmentation is widely used in image analysis, where the goal is to identify relevant features from a corpus of features that also have irrelevant differences.
+To understand data augmentation, imagine pathology slides that have been scanned.
+Each slide may be prepared and scanned in a subtly different orientation depending on who generated the data or at somewhat different magnifications depending on the center that performed the imaging.
+A deep learning method may learn to identify these subtle differences, which is undesirable.
+It's also possible that there are simply too few slides to train a deep learning algorithm.
+To address these challenges, deep learning practitioners can apply arbitrary rotations, zooms, and other irrelevant transformations to image data during training.
+This process is called data augmentation.
+
+No data augmentation approaches for genomic data have been published to date.
+However, we expect that very fast abundance estimates [@doi:10.1038/nmeth.4197 @10.1038/nbt.3519] will make a data augmentation approach feasible in this domain as well.
```

Comment:
used `data` in case @rob-p comes up with a way that goes from an intermediate data structure